Generals/Code/Main/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 add_executable(g_generals WIN32)
 
-set_target_properties(g_generals PROPERTIES OUTPUT_NAME generals)
+# Use a binary name that doesn't conflict with original game.
+if("${CMAKE_SYSTEM}" MATCHES "Windows")
+    set_target_properties(g_generals PROPERTIES OUTPUT_NAME OpenGenerals)
```

Comment:
We did not yet settle on OpenGenerals. Original generals.exe does not begin with uppercase.

Until we figure out better name, perhaps lets just do:

```
generalsV
generalsZH
WorldBuilderV
WorldBuilderZH
```

V = vanilla
ZH = zero hour